src/content/docs/en/starter-server.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Starter Server"
+description: "A resource to easily create a Basic Java Edition Minecraft server"
+---
+
+import {COMMUNITY_INVITE_URL} from '../../../consts';
+
+Starter Server is a repository that contains a basic Java Edition Minecraft server. The server here is not intended to
+be the next Hypixel out of the box; instead, it should be seen as an easy way to get an up-to-date Minecraft server
+quickly, to play with some friends in a home-hosted or at least self-hosted environment.
+
+Since it's not intended to be fantastic or comprehensive, it contains just a few plugins:
+
+- LuckPerms
+- EssentialsX
+- EssentialsXAntiBuild
+- EssentialsXChat
+- Vault
+
+## 🔗 Get Started
+
+You can find the instructions to get started in minutes here:
+https://github.com/syscraft-mc/starter-server#how-to-get-started
+
+You can also find common issues and frequently asked questions here:
+https://github.com/syscraft-mc/starter-server#troubleshooting-and-faq
+
+Should you get stuck at any point, feel free to ask the community on our
+<a href={COMMUNITY_INVITE_URL}>Discord server</a>!
```

Comment:
Shouldn't this use MD syntax?

Reply:
From what I could tell, the only way to use the variable in MDX was like this. Simply doing the `[]()` syntax did not allow me to add it with any format I tried.